The database HSQLDB is located at /usr/share/java/hsqldb.jar in Debian system
--- a/build.xml
+++ b/build.xml
@@ -2,7 +2,7 @@
 <project basedir="." default="jar" name="jcathy">
 	<property name="version" value="0.7.4"/>
 	
-	<property name="lib" value="lib"/>
+	<property name="lib" value="/usr/share/java"/>
 	<property name="src" value="src"/>
 	<property name="bin" value="bin"/>
 	
@@ -85,6 +85,5 @@
 		</jar>
 		<echo message="Completed!"/>
 		<echo message="${ant.project.name} has been built and packaged into jar file"/>
-		<echo message="'${lib}' directory and its content must be present in the same folder of ${jarname} file"/>
 	</target>
 </project>
